bin/verilator

```diff
@@ -4494,6 +4538,10 @@ With this change this assignment needs to be evaluated only once.  These
 sort of changes may also speed up your traditional event driven simulator,
 as it will result in fewer events per cycle.
 
+Another way to resolve this warning is to add split_var meta comment
+explained above. The variable will be split internnaly so that Verilator
+can optimize well.
```

Comment:
Another way to resolve this warning is to add the split_var meta comment
explained above. This will cause the variable to be split internally so that Verilator
can better optimize it.